src/components/app/SingleStrategy/GenLender.tsx

```diff
@@ -130,16 +136,16 @@ export const GenLender = (props: GenLenderProps) => {
                 value: estimateAdjustPositionLowestAPR,
             },
             {
-                key: 'Potential:',
+                key: 'Potential APR:',
                 value: estimateAdjustPositionPotential,
             },
         ];
-        return <CardContent data={data} key={strategy.address} />;
+        return <CardContent data={cardData} key={strategy.address} />;
     };
 
     return (
         <>
-            {isLoading ? (
+            {loading ? (
```

Comment:
I think we should not nest ternary operators, it makes reading a little bit hard.

Could we return based on the state instead

```typescript
if (loading) {
    return <StyledTypography>Loading...</StyledTypography>
}

if (error) {
    return <ErrorAlert message={error} />
}

// ...
```

We should also check for `loading && !data` rather than just `loading`, as we might not want to show loading when we are fetching results on the background. So we would only want to show the loading until we've got some data. This might be useful if in we have a `refetch`

Reply:
Yeah I opened #208 to address this, but I can change it now